mail/views.py

```diff
@@ -237,3 +239,34 @@ def post(self, request, *args, **kwargs):
             status=mailgun_response.status_code,
             data=generate_mailgun_response_json(mailgun_response)
         )
+
+
+class EmailBouncedView(APIView):
+    """
+    View class that handles HTTP requests to course team mail API
+    """
+    permission_classes = (MailGunWebHookPermission, )
+
+    def post(self, request, *args, **kargs):  # pylint: disable=unused-argument
+        """
+        POST method handler
+        """
+        event = request.POST.get("event", None)
+        recipient = request.POST.get("recipient", None)
+        error = request.POST.get("error", None)
+        message_headers = request.POST.get("message-headers", "NA")
+        log_error_on_bounce = request.POST.get("log_error_on_bounce", False)
```

Comment:
This comes from the header, not the POST body. See the docs here: https://documentation.mailgun.com/en/latest/user_manual.html#tracking-bounces

Reply:
from http://bin.mailgun.net/2dc22ec1 it seems like it parse custom variables

Reply:
I see, yes, you're right